core/src/test/java/org/apache/any23/vocab/RDFSchemaUtilsTest.java

```diff
@@ -53,7 +53,7 @@ public void testSerializeVocabulariesNTriples() {
      */
     @Test
     public void testSerializeVocabulariesRDFXML() {
-        serializeVocabularies(RDFFormat.RDFXML, 4987); // Effective lines + separators.
+        serializeVocabularies(RDFFormat.RDFXML, 5252); // Effective lines + separators. //4992
```

Comment:
I have now removed the comment. I was not sure about reason behind this test failure at that time, so added a comment for the future reference, later understood that this is due to changes related to added vocabularies by me.